src/components/common/PaginatedTxns/index.tsx

```diff
@@ -8,52 +7,64 @@ import useTxQueue from '@/hooks/useTxQueue'
 import PagePlaceholder from '../PagePlaceholder'
 import LoadMoreButton from '../LoadMoreButton'
 import SkeletonTxList from './SkeletonTxList'
-import { TX_LIST_EVENTS } from '@/services/analytics/events/txList'
-import { trackEvent } from '@/services/analytics/analytics'
+
+const NoQueuedTxns = () => (
+  <Box mt="5vh">
+    <PagePlaceholder imageUrl="/images/no-transactions.svg" text="Queued transactions will appear here" />
+  </Box>
+)
+
+const TxPage = ({
+  pageUrl,
+  useTxns,
+  onNextPage,
+}: {
+  pageUrl?: string
```

Comment:
```suggestion
  pageUrl: string
```

It looks like `pageUrl` always exists